<launch>
  <arg name="stage" default="2"/>
  <param name="stage_number"  value="$(arg stage)"/>
  <node pkg="turtlebot3_dqn" type="client_local_train.py" name="client_local_train" output="screen" />
</launch>
